openmetadata-docs/content/deployment/upgrade/bare-metal.md

```diff
@@ -92,7 +174,22 @@ binaries. You may restart the server by running the following command.
 ./bin/openmetadata.sh start
 ```
 
-### Optional - Upgrade all your connectors
+### 8. Reindex ElasticSearch
+
+We have added a conditional suggestion mapping for all of the elasticsearch indexes. This may require re-indexing. With 0.12.1 its never been easier to index your metadata
+
+#### 8.1 Go to Settings -> Event Publishers -> ElasticSearch
+
+<Image src="/images/deployment/upgrade/elasticsearch-re-index.png" alt="create-project" caption="Create a New Project"/>
+
+#### 8.2 Make sure you select "Recreate Indexes"
+
+Click on the "Recreate Indexes" lable and click "Re Index All"
+
+
+
+
+###  Upgrade all your connectors
```

Comment:
Should we at the end link to the 0.11 to 0.12 page for users to follow how to upgrade Airflow?